### ISSUE TITLE
Saved query executing differently in different projects/windows

### ISSUE THREAD

**Issue by anguspmitchell** [TeamCodeStream/codestream#713]
Query is exactly the same in both cases. I've tried refreshing and dropping + re-adding.

### Executing Correctly
```
is:pr is:open repo:xxx-org/yyyy-app -reviewed-by:@me -author:@me base:develop created:>2021-05-15 
```
![Screen Shot 2021-09-09 at 4 06 55 PM](https://user-images.githubusercontent.com/4749149/132762733-0544b99b-eea7-4a1c-8982-f64d5f19197c.png)

### Fetching from the wrong repo
```
is:pr is:open repo:xxx-org/yyyy-app -reviewed-by:@me -author:@me base:develop created:>2021-05-15 
```
![Screen Shot 2021-09-09 at 4 06 42 PM](https://user-images.githubusercontent.com/4749149/132762735-a76b21b0-9796-421c-b147-de4d5fdb7a9f.png)

**Comment by planteater** [comment]
@anguspmitchell - Could I trouble you for some clarification on what you mean by "different projects/windows"? The query is the same, just with different repos specified?

Ultimately, we'll probably need a log file to dig into this. Reproduce the issue and then open the Output view in VS Code (“View: Toggle Output” from the command palette) and select "CodeStream (Agent)" from the dropdown menu at the top-right. Copy all of the output you see and save it in a text file, then send it to me at support@codestream.com. Thanks!

**Comment by planteater** [comment]
Hi @anguspmitchell - Just wanted to follow up on the above. Thanks!

**Comment by anguspmitchell** [comment]
I need those as negated terms -- and that part of it seems to work FWIW. Is
there another to negate certain categories?

**Comment by planteater** [comment]
@anguspmitchell - I probably should have thought of this much sooner, but it occurrs to me that you've got the option selected to "Only show PRs from open repos" in the Pull Requests section (see gear menu). Might that explain why you're seeing different results in different project windows?

**Comment by anguspmitchell** [comment]
I don't think so, since I have both queries set up across all of VS Code

*Window 1 - SPA Project*
- query for SPA repo
- query for API repo

*Window 2 - API Project*
- query for SPA repo
- query for API repo

If the SPA query only worked in the SPA window and the API query only
worked in the API window, that'd be fine. But what's happening is that the
SPA query in the SPA window is fetching from both repositories.

**Comment by planteater** [comment]
@anguspmitchell - I know it's been a long time coming, but we finally have a fix for this ready for out next release. Should be early next week. I should note that this issue does appear to only happen when you have the option checked to "Only show PRs from open repos". If you uncheck that things should be fine. But our fix will make things work even when that's checked. I'll update you as soon as the fix is available.

![Screen Shot 2022-01-28 at 11 34 30 AM](https://user-images.githubusercontent.com/3442587/151585598-c0126fd9-a489-4480-a17b-1d77ddac2b53.png)

**Comment by anguspmitchell** [comment]
Thanks!

BTW, unchecking that box does "fix" the problem, although that would not
have been clear to me without your help

**Comment by planteater** [comment]
@anguspmitchell - This fix is now available in v12.8. Thanks for your patience!
